src/components/manager.ts

```diff
@@ -10,9 +10,9 @@ import {InputFileRegExp} from '../utils/inputfilepath'
 
 import type {Extension} from '../main'
 import * as eventbus from './eventbus'
-import type {Suggestion as CiteEntry} from '../providers/completer/citation'
-import type {Suggestion as CmdEnvEntry} from '../providers/completer/command'
-import type {Suggestion as GlossEntry} from '../providers/completer/glossary'
+import type {CmdEnvSuggestion as CmdEnvEntry} from '../providers/completer/command'
+import type {CiteSuggestion as CiteEntry} from '../providers/completer/citation'
+import type {GlossarySuggestion as GlossaryEntry} from '../providers/completer/glossary'
```

Comment:
What is the reason for using aliases here? We would have no reason to use the aliases.